__tests__/integrations/FacebookPixel/FacebookPixel.test.js

```diff
@@ -32,13 +32,10 @@ describe('FacebookPixel init tests', () => {
     facebookPixel.init();
     expect(typeof window.fbq).toBe('function');
     expect(facebookPixel.userPayload).toStrictEqual({
-      db: undefined,
```

Comment:
Are these changes expected?

Reply:
yes, sending undefined will not make any sense to destination.